src/models/roles.js

```diff
@@ -0,0 +1,43 @@
+import {Model} from 'sequelize';
+
+module.exports = (sequelize, DataTypes) => {
+  // eslint-disable-next-line require-jsdoc
+  class Roles extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+     static associate({Permissions}) {
+      this.hasMany(Permissions, { foreignKey: "roleId" });
```

Comment:
I think we have many_to_many relationships on creating roles and permissions, though you have used one_to_many. check well the Trello board here https://trello.com/c/zRlfxZP8/9-create-roles-and-set-permissions